app/assets/stylesheets/sections/deploy.scss

```diff
@@ -0,0 +1,56 @@
+$dropbox_color: #1081de;
+$zip_color:     #a58463;
+
+.nav-upload {
+
+  .tab-description {
+    margin-left: 10px;
+
+    .title {
+      font-size: 14px;
+      font-weight: bold;
+    }
+  }
+
+  .fa {
+    font-size: 42px;
+  }
+
+  li {
+    a:hover {
+      background-color: inherit;
+      border-color: transparent;
+
+      .fa-dropbox {
+        color: $dropbox_color;
+      }
+
+      .fa-file-archive-o {
+        color: $zip_color;
+      }
+    }
+
+    .fa-dropbox {
+      color: lighten($dropbox_color, 15);
+    }
+
+    .fa-file-archive-o {
+      color: lighten($zip_color, 15);
+    }
+  }
+
+  li.active {
+    a, a:hover {
+      background-color: inherit;
+      border-color: $gray-light;
+    }
+
+    .fa-dropbox {
+      color: $dropbox_color;
+    }
+
+    .fa-file-archive-o {
+      color: $zip_color;
```

Comment:
Name of variable `zip_color` should be written in all lowercase letters with hyphens instead of underscores